libs/api/libbpf_link.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "api_internal.h"
+#pragma warning(push)
+#pragma warning(disable : 4200)
+#include "libbpf.h"
+#pragma warning(pop)
+#include "libbpf_internal.h"
+
+int
+bpf_link__pin(struct bpf_link* link, const char* path)
+{
+    int err;
+
+    if (link->pin_path)
+        return -EBUSY;
```

Comment:
```suggestion
        return libbpf_err(-EBUSY);
```
Call libbpf_err to set errno on error.